src/weather_magic/shaders.cljs

```diff
@@ -65,7 +70,8 @@
               :lightDir   [:vec3 [1 0 1]]
               :lightCol   [:vec3 [1 1 1]]
               :ambientCol [:vec3 [0 0 0.1]]
-              :frameCounter [:int 0]}
+              :year       [:float 0.0]
```

Comment:
Du lägger in year och range som uniforms i alla shaders uniforms fast de bara används i temperature-shadern? Det är klart vi ska använda dem i alla shaders sen, men vi gör det ju inte ännu. Så det kanske är ok